Source/WebCore/Modules/WebGPU/GPUDevice.h

```diff
@@ -75,7 +75,7 @@ class GPUSupportedLimits;
 class GPUTexture;
 struct GPUTextureDescriptor;
 
-class GPUDevice : public RefCounted<GPUDevice>, public ActiveDOMObject, public EventTargetWithInlineData {
```

Comment:
For simplicity, why don't we alias EventTargetWithInlineData with EventTarget.
Once this patch landed, we can clean that up as a follow up.

Reply:
Since this is mechanically changeable, this patch also just changed all EventTargetWithInlineData with EventTarget.